examples/threathunter/threat_intelligence/stix_taxii.py

```diff
@@ -12,11 +15,15 @@
 from feed_helper import FeedHelper
 from datetime import datetime
 from results import AnalysisResult
+from cbapi.psc.threathunter.models import Feed
 import urllib3
+import copy
 
-logging.basicConfig(filename='stix.log', filemode='w', level=logging.DEBUG)
-
-handled_exceptions = (NoURIProvidedError, ClientException)
+# logging.basicConfig(filename='stix.log', filemode='w', level=logging.DEBUG)
+logging.basicConfig(filename='stix.log', filemode='w', format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+    datefmt='%Y-%m-%d:%H:%M:%S',
+    level=logging.DEBUG)
```

Comment:
Leaving this at DEBUG will be a little log heavy since this will log out all cbapi debug logs too. You may want to make this attached to a command argument as well?

Reply:
I've updated this to be INFO once more. The idea of adding a command argument for logging levels is a great idea. I think we did this on the original cb-taxii-connector that this is based on. To reduce the delay in getting the bug fixes to customers waiting, I'd like to push that feature to another ticket, but it shouldn't be too hard to implement once we start it.